sqlitewrap/SysLogs.cpp

```diff
@@ -15,7 +15,7 @@
 #include "Database.h"
 #include "Query.h"
 #include "IError.h"
-#include "SysLog.h"
+#include "SysLogs.h"
```

Comment:
Syslog was a header that was already in use on some platform.  So this had to be changed for compatibility.